### PR TITLE
Kill all CLI aliases.

### PR DESCRIPTION
**Pull request by EvanJRichard**
## Summary
Grep for aliases in the cli and remove.

## Testing Plan
`yarn test` successful.

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ironfish-rust-nodejs/index.d.ts [hunk 1]
```diff
@@ -3,12 +3,6 @@
 
 /* auto-generated by NAPI-RS */
 
-export class ExternalObject<T> {
-  readonly '': {
-    readonly '': unique symbol
-    [K: symbol]: T
-  }
-}
```
Comment (NullSoldier) [inline ironfish-rust-nodejs/index.d.ts:11]: Not sure why the build re-generated this, can we revert this? I'll ask rohan why this is being modified? Maybe this is a rust version thing?

File: ironfish-rust-nodejs/index.js [hunk 2]
```diff
@@ -11,7 +11,8 @@ function isMusl() {
   // For Node 10
   if (!process.report || typeof process.report.getReport !== 'function') {
     try {
-      return readFileSync('/usr/bin/ldd', 'utf8').includes('musl')
+      const lddPath = require('child_process').execSync('which ldd').toString().trim();
+      return readFileSync(lddPath, 'utf8').includes('musl')
```
Comment (NullSoldier) [inline ironfish-rust-nodejs/index.js:15]: some more generated code to revert for now I think

File: .yarnrc.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (NullSoldier) [inline .yarnrc.yml:1]: What does this do?

**Comment by EvanJRichard** [comment]
`yarn test` output I'd like help interpreting:

```
lerna info Executing command in 3 packages: "yarn run test"
eslint-plugin-ironfish: node:internal/process/esm_loader:97
eslint-plugin-ironfish:     internalBinding('errors').triggerUncaughtException(
eslint-plugin-ironfish:                               ^
eslint-plugin-ironfish: AssertionError [ERR_ASSERTION]: Should have 1 error but had 0: []
eslint-plugin-ironfish:     at testInvalidTemplate (/Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:718:24)
eslint-plugin-ironfish:     at Function.<anonymous> (/Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:952:29)
eslint-plugin-ironfish:     at Function.itDefaultHandler (/Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:299:23)
eslint-plugin-ironfish:     at /Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:949:63
eslint-plugin-ironfish:     at Array.forEach (<anonymous>)
eslint-plugin-ironfish:     at Function.<anonymous> (/Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:948:30)
eslint-plugin-ironfish:     at Function.describeDefaultHandler (/Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:316:19)
eslint-plugin-ironfish:     at Function.<anonymous> (/Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:947:24)
eslint-plugin-ironfish:     at Function.describeDefaultHandler (/Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:316:19)
eslint-plugin-ironfish:     at RuleTester.run (/Users/evan/Documents/GitHub/ironfish/.yarn/cache/eslint-npm-7.29.0-aec1968387-8a18377c76.zip/node_modules/eslint/lib/rule-tester/rule-tester.js:935:20) {
eslint-plugin-ironfish:   generatedMessage: false,
eslint-plugin-ironfish:   code: 'ERR_ASSERTION',
eslint-plugin-ironfish:   actual: 0,
eslint-plugin-ironfish:   expected: 1,
eslint-plugin-ironfish:   operator: 'strictEqual'
eslint-plugin-ironfish: }
```

File: .yarnrc.yml (hunk 3 above)
Comment (EvanJRichard) [reply .yarnrc.yml:1]: Not sure, more autogenerated code I think, can revert

File: ironfish-rust-nodejs/index.d.ts (hunk 1 above)
Comment (EvanJRichard) [reply ironfish-rust-nodejs/index.d.ts:11]: Can revert, will leave it as-is til tomorrow

**Comment by NullSoldier** [comment]
That output looks strange. Try running it individually in each of the repos to see which one is causing it. It looks like you're running that at the root. You can try a yarn clean, and yarn build at the root too.

**Comment by EvanJRichard** [comment]
- `yarn test` now successful, not sure exactly what I did so I mostly did a lot of cleanup and reverting til things were fixed
- is `master` the appropriate target branch here?

**Comment by NullSoldier** [comment]
This should target staging. Our general flow is`kill-aliases` -> `staging` -> `master` (Stable Release). This is because a lot of people pull down the default branch and auto run nodes off it so we couldn't have changes that require breaking build changes there without announcement to the community first.

**Comment by EvanJRichard** [comment]
Target branch changed.

File: ironfish-cli/package.json [hunk 4]
```diff
@@ -108,9 +108,7 @@
       }
     }
   },
-  "bin": {
-    "ironfish": "./bin/run"
-  },
```
[the remark this answers is not in the document]
Comment (NullSoldier) [reply ironfish-cli/package.json:113]: yep

**Comment by EvanJRichard** [comment]
Moved to #2825

**Pull request closed without merging**